src/test/java/com/salesforce/dynamodbv2/dynamodblocal/AmazonDynamoDbLocal.java

```diff
@@ -56,26 +56,19 @@ private static DynamoDbClients getNewAmazonDynamoDbLocalWithStreams() {
         AmazonDynamoDBLocal amazonDynamoDbLocalClient = getNewAmazonDynamoDbLocalClient();
         localAmazonDynamoDb = amazonDynamoDbLocalClient.amazonDynamoDB();
         localAmazonDynamoDbStreams = amazonDynamoDbLocalClient.amazonDynamoDBStreams();
-        return new DynamoDbClients(localAmazonDynamoDb, localAmazonDynamoDbStreams);
+        return new DynamoDbClients(localAmazonDynamoDb);
     }
 
     static class DynamoDbClients {
         private final AmazonDynamoDB amazonDynamoDb;
-        private final AmazonDynamoDBStreams amazonDynamoDbStreams;
 
-        DynamoDbClients(AmazonDynamoDB amazonDynamoDb,
-            AmazonDynamoDBStreams amazonDynamoDbStreams) {
+        DynamoDbClients(AmazonDynamoDB amazonDynamoDb) {
             this.amazonDynamoDb = amazonDynamoDb;
-            this.amazonDynamoDbStreams = amazonDynamoDbStreams;
         }
 
         AmazonDynamoDB getAmazonDynamoDb() {
             return amazonDynamoDb;
         }
-
-        AmazonDynamoDBStreams getAmazonDynamoDbStreams() {
-            return amazonDynamoDbStreams;
-        }
```

Comment:
Unused declaration

Reply:
why was the `amazonDynamoDbStreams` field in there to begin with?

Reply:
I don't recall.  👎

Reply:
I'm a bit wary to delete this then—perhaps @busjaeger recalls/knows?